docker/runtime/python-2.7/kubeless.py

```diff
@@ -1,31 +1,24 @@
 #!/usr/bin/env python
 
-import sys
 import os
 import imp
 
-from bottle import route, run, request
+import bottle
```

Comment:
The whole module gets loaded either way (see contents of sys.modules, for example).  This seems to explain it as well as I could here: https://softwareengineering.stackexchange.com/questions/187403/import-module-vs-from-module-import-function/187471